docs/usage.md

```diff
@@ -167,9 +149,9 @@ A valid example of a `phenotype.csv` file (matching the TCGA example input CSV f
 
 `nf-core/circrna` provides 3 analysis modules to the user:
 
-1. circRNA quantification & annotation.
-2. miRNA target prediction.
-3. Differential circRNA expression analysis.
+1. BSJ detection
+2. Common quantification of circular and linear transcriptome
```

Comment:
What does "common quantification" mean?

Reply:
Do you thing "Joint quantification" is clear enough? 
It should basically mean that there is a run of kallisto (actually a wrapper called psirc-quant), which takes a single fasta file containing both the known linear and detected circular transcriptome.